realm/realm-library/src/androidTest/java/io/realm/rule/TestRealmConfigurationFactory.java

```diff
@@ -151,4 +152,17 @@ public void copyRealmFromAssets(Context context, String realmPath, String newNam
         outputStream.close();
         is.close();
     }
+
+    public static void emulateRxJavaUnavailable(RealmConfiguration config) {
```

Comment:
Any reason this method is on `TestRealmConfigurationFactory` instead of `TestHelper`? This functionality doesn't really seem to belong in this class?